Rakefile

```diff
@@ -11,17 +11,25 @@ def brew_install(package, *args)
     # brew did not error out, verify tmux is greater than 1.8
     # e.g. brew_tmux_query = 'tmux 1.9a'
     installed_version = versions.split(/\n/).first.split(' ')[1]
-    unless version_match?(options[:version], installed_version)
+    unless version_match?(options[:requires], installed_version)
       sh "brew upgrade #{package} #{args.join ' '}"
     end
   end
 end
 
-def version_match?(requirement, version)
+def version_match?(requirement, v)
   # This is a hack, but it lets us avoid a gem dep for version checking.
+  version = parse_version(v)
```

Comment:
`gem --version` gives me 2.4.6. I think it's got something to do with my ruby version. 2.1.3 works fine, but 2.2.0 and 2.2.1 fail.
![screen shot 2015-05-07 at 5 54 27 pm](https://cloud.githubusercontent.com/assets/973173/7528776/752d0b4e-f4e2-11e4-9e1d-f34c0caf16ce.png)

![screen shot 2015-05-07 at 6 02 20 pm](https://cloud.githubusercontent.com/assets/973173/7528831/7087b066-f4e3-11e4-875a-e84a47c3aead.png)
![screen shot 2015-05-07 at 5 55 09 pm](https://cloud.githubusercontent.com/assets/973173/7528775/752c5708-f4e2-11e4-9a55-9775bede62d9.png)